api/api/settings_quick_test.py

```diff
@@ -1,14 +1,15 @@
+import os
+
 # noinspection PyUnresolvedReferences
 from api.settings import *
 
 # noinspection PyUnresolvedReferences
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': 'desecapi.sqlite',
-        'TEST': {
-            'CHARSET': 'utf8mb4',
-        },
+        'ENGINE': 'django.db.backends.postgresql',
+        'NAME': 'desec',
+        'USER': 'desec',
+        'HOST': '127.0.0.1' if os.environ.get('DJANGO_TEST', '') == '1' else 'dbapi',
```

Comment:
Is this variable name defined and used by django? Otherwise is should be prefixed with DESECSTACK

Reply:
done